crates/env/src/engine/on_chain/ext.rs

```diff
@@ -610,6 +616,20 @@ pub fn rent_params(output: &mut &mut [u8]) {
     extract_from_slice(output, output_len as usize);
 }
 
+pub fn rent_status(at_refcount: Option<u32>, output: &mut &mut [u8]) {
```

Comment:
Maybe `core::num::NonZeroU32` could be used to express that `0` is not a valid input.